### ISSUE TITLE
Mp3s are buggy

### ISSUE THREAD

**Issue by dekrypted** [Gucci637/Fentanyl#6]
I know you already said they are unstable but MP3's dont work for me, how can I make them work?

**Comment by Gucci637** [comment]
Yeah they are not super stable. Try this new release it's fixed (I think)
